lib/mac-learning.c

```diff
@@ -24,6 +24,7 @@
 #include "coverage.h"
 #include "hash.h"
 #include "openvswitch/list.h"
+#include "openvswitch/ovs-p4rt.h"
```

Comment:
line 35 already includes this file and also it should be wrapped under P4OVS supported checks?

Reply:
Nuts! You're right.